spec/codelog/command/step/version_spec.rb

```diff
@@ -120,4 +120,17 @@
       described_class.run '1.2.3', '2012-12-12'
     end
   end
+
+
+  describe '#changes_hash' do
+    context "when a not parseable yml file is given" do
```

Comment:
I'm just nitpicking here, but I think it should say "non parseable" instead of "not parseable"

Reply:
Change was made.